content/millennium/r4/clinical/request-and-response/communication.md

```diff
@@ -39,41 +45,41 @@ All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/St
 
 ID             | Value\[x] Type      | Description
 ---------------|---------------------|----------------------------------------------------------------------------------
-`reply-to`     | [`Reference`]       | Used to direct where replies to the communication should be sent. Must be a Group.
-`email-status` | [`CodeableConcept`] | Status of the electronic communication (in-progress, completed, received).
+`reply-to`     | [`Reference`]       | Directs where replies to the communication are sent. Must be a group.
+`email-status` | [`CodeableConcept`] | The status of the electronic communication (in-progress, completed, or received).
```

Comment:
I forget, have we been listing the required values here or in the notes section?